docs/basics/101-180-FAQ.rst

```diff
@@ -411,6 +411,21 @@ dataset.
 When using the terminal command ``cp`` [#f1]_, it is sufficient to use the
 ``-L``/``--dereference`` option. This will follow symbolic links, and make
 sure that content gets moved instead of symlinks.
+Remember that if you are copying some annexed content out of a dataset without
+unlocking it first, you will only have "read" access on the files you have just 
+copied. Therefore you can :
+- either unlock the files before copying them out, 
+- or copy them and then use the command ``chmod`` to be able to edit the file.
+
+```
+# this will give you 'write' permission on the file
+chmod +w filename
+```
```

Comment:
This how to get the code markup in RST:

```suggestion
.. code-block:: bash

   # this will give you 'write' permission on the file
   $ chmod +w filename
```

I would add a $ at the start of the command line for consistency across the book :)

Reply:
Awww. Look at past me dumping markdown in a reStructured text document. 

But now... I know ~kung-fu~ RST. Well... I have been teaching myself for the past 36 hours.

![MarvelousBitterIrishwaterspaniel-size_restricted](https://user-images.githubusercontent.com/6961185/97332566-e14e1680-187a-11eb-8bcb-044c863facd5.gif)